portal/views/assessment_engine.py

```diff
@@ -1095,16 +1096,21 @@ def present_assessment(instrument_id):
         description: if missing valid OAuth token or bad `next` parameter
 
     """
+    AE_intervention = Intervention.query.filter_by(name='assessment_engine').one()
+
     # Todo: replace with proper models
     instruments = current_app.config['INSTRUMENTS']
     clients_instruments = current_app.config['CLIENTS_INSTRUMENTS']
 
     if not instrument_id or not instrument_id in instruments:
         abort(404, "No matching assessment found: %s" % instrument_id)
 
-    for client_id, instrument_urls in clients_instruments.items():
-        if instrument_id in instrument_urls:
-            assessment_url = instrument_urls[instrument_id]
+    for client_id, instruments in clients_instruments.items():
+        if instrument_id in instruments and client_id == AE_intervention.client_id:
```

Comment:
This seems a little more fragile than I believe we need.  Why duplicate locations for setting the client_id of the assessment engine?  It lives in the database tables, do we also need to set it in the config (as I see in staging):
`ASSESSMENT_ENGINE_CLIENT_ID = 'UVAh1naZO8tztccxb98yj4XRSanAr8Zhzu4Zx5Kc'`

If we do need such flexibility, should it move to the database?